sr_gui_change_controllers/src/sr_gui_change_controllers/change_controllers.py

```diff
@@ -71,60 +72,88 @@ def populate_controllers(self):
-                                                                                                          for hand_joint_prefix in hand_joint_prefixes],
-                            "mixed": ["sh_{0}{1}_mixed_position_velocity_controller".format(hand_joint_prefix, joint) for joint in joints 
-                                                                                                          for hand_joint_prefix in hand_joint_prefixes],
-                            "velocity": ["sh_{0}{1}_velocity_controller".format(hand_joint_prefix, joint) for joint in joints 
-                                                                                                          for hand_joint_prefix in hand_joint_prefixes],
+        self.controllers = {
+            "effort": [
+                "sh_{0}{1}_effort_controller".format(hand_joint_prefix, joint)
+                for joint in joints for hand_joint_prefix in
+                hand_joint_prefixes],
+            "position": [
+                "sh_{0}{1}_position_controller".format(hand_joint_prefix,
+                                                       joint)
+                for joint in joints for hand_joint_prefix in
+                hand_joint_prefixes],
+            "mixed": [
+                "sh_{0}{1}_mixed_position_velocity_controller".format(
+                    hand_joint_prefix, joint)
+                for joint in joints for hand_joint_prefix in
+                hand_joint_prefixes],
+            "velocity": [
+                "sh_{0}{1}_velocity_controller".format(
+                    hand_joint_prefix, joint)
+                for joint in joints for hand_joint_prefix in
+                hand_joint_prefixes],
                             "stop": []}
 
-        self.managed_controllers = [cont for type_conts in self.controllers.itervalues() for cont in type_conts]
-
+        self.managed_controllers = [
+            cont for type_conts in self.controllers.itervalues()
+            for cont in type_conts]
 
     def __init__(self, context):
         super(SrGuiChangeControllers, self).__init__(context)
         self.setObjectName('SrGuiChangeControllers')
 
         self._publisher = None
-        
+
         self._widget = QWidget()
 
-        ui_file = os.path.join(rospkg.RosPack().get_path('sr_gui_change_controllers'), 'uis', 'SrChangeControllers.ui')
+        ui_file = os.path.join(
+            rospkg.RosPack().get_path('sr_gui_change_controllers'), 'uis',
+            'SrChangeControllers.ui')
         loadUi(ui_file, self._widget)
         self._widget.setObjectName('SrChangeControllersUi')
         context.add_widget(self._widget)
-        
-        #setting the prefixes
+
+        # setting the prefixes
+        self.hand_finder = HandFinder()
+        hand_finder = HandFinder()
+        hand_parameters = hand_finder.get_hand_parameters()
+
         self._prefix = ""
-        
         self._widget.select_prefix.addItem("")
```

Comment:
I think we should remove the empty one, as currently the sr_config only works for `rh` or `lh`.